samples/storage_csm_powermax_v2101.yaml

```diff
@@ -208,17 +208,17 @@ spec:
         - key: "node.kubernetes.io/network-unavailable"
           operator: "Exists"
           effect: "NoExecute"
-      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
-      # - key: "node-role.kubernetes.io/master"
-      #   operator: "Exists"
-      #   effect: "NoSchedule"
-      # Uncomment if CSM for Resiliency and CSI Driver pods monitor is enabled
-      #  - key: "offline.powermax.storage.dell.com"
-      #    operator: "Exists"
-      #    effect: "NoSchedule"
-      #  - key: "powermax.podmon.storage.dell.com"
-      #    operator: "Exists"
-      #    effect: "NoSchedule"
+        # Uncomment and left-tab if nodes you wish to use have the node-role.kubernetes.io/master taint
```

Comment:
one yaml warning here, but the indenting looks legit, is there a way to quiet this or something? or is something actually off?

Reply:
I've been fighting with that one; you can see the comment advising a left-tab because of it, I tried to shift it right because it wasn't happy about the comment placement. 
I really don't know what the linter wants, here. It's not failing the check, it's just a warning, but I'd still like it fixed. If you have any suggestions, let me know, because it isn't happy with the comments here and it isn't happy with them two spaces to the left, and four spaces to the left looks even worse.